src/components/AddToEmbeddedPluginButton/AddToEmbeddedPluginButton.vue

```diff
@@ -111,24 +114,39 @@ async function onConfirmedToAdd() {
   }
 
   if (elevatorCallbackType.value === "lti") {
-    const data = await api.postLtiPayload({
-      fileObjectId: props.fileHandlerId,
-      excerptId: "",
-      returnUrl: returnUrl.value ?? "",
-    });
-
-    ltiContentItems.value = JSON.stringify(data);
-
-    nextTick(() => {
-      if (!returnForm.value) {
-        addingToPluginStatus.value = "error";
-        throw new Error("Return form not set");
-      }
-
-      returnForm.value.submit();
-      addingToPluginStatus.value = "success";
-      toastStore.addToast({ message: "Added to Canvas" });
-    });
+
+
+    if (elevatorLTIVersion.value == "1.3") {
+      const data = await api.postLtiPayload13({
+        fileObjectId: props.fileHandlerId,
+        excerptId: "",
+        returnUrl: returnUrl.value ?? "",
+        launchId: elevatorLaunchId.value ?? "",
+        userId: userId.value ?? "",
+      });
+
+      document.body.innerHTML += data;
+      (document.getElementById('auto_submit') as HTMLFormElement)?.submit();
```

Comment:
Sorry – I'm not following this part. What's happening here? Is this something specific to how LTI1.3 works?

Reply:
yeah, more or less - because DeepLinking happens inside an iframe, we need to get the client itself to make the deep linking submission to the LMS (so the LMS can then close the iframe/do whatever). So it can't be server<->server like grade passback is with LTI1.3. 

The specific auto_submit ID name comes from the LTI library we use.

Reply:
ah, that makes sense. Also why I couldn't find the `auto_submit` id anywhere in the elevator or ui code. Thanks!